impexp-client/src/main/java/org/citydb/cli/options/deleter/QueryOption.java

```diff
@@ -0,0 +1,175 @@
+/*
+ * 3D City Database - The Open Source CityGML Database
+ * http://www.3dcitydb.org/
+ *
+ * Copyright 2013 - 2020
+ * Chair of Geoinformatics
+ * Technical University of Munich, Germany
+ * https://www.gis.bgu.tum.de/
+ *
+ * The 3D City Database is jointly developed with the following
+ * cooperation partners:
+ *
+ * virtualcitySYSTEMS GmbH, Berlin <http://www.virtualcitysystems.de/>
+ * M.O.S.S. Computer Grafik Systeme GmbH, Taufkirchen <http://www.moss.de/>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.citydb.cli.options.deleter;
+
+import org.citydb.config.project.query.QueryConfig;
+import org.citydb.config.project.query.filter.selection.AbstractPredicate;
+import org.citydb.config.project.query.filter.selection.SelectionFilter;
+import org.citydb.config.project.query.filter.selection.id.DatabaseIdOperator;
+import org.citydb.config.project.query.filter.selection.id.ResourceIdOperator;
+import org.citydb.config.project.query.filter.selection.logical.AndOperator;
+import org.citydb.config.project.query.filter.selection.spatial.AbstractSpatialOperator;
+import org.citydb.config.project.query.filter.selection.sql.SelectOperator;
+import org.citydb.plugin.cli.CliOption;
+import org.citydb.plugin.cli.CounterOption;
+import org.citydb.plugin.cli.DatabaseIdOption;
+import org.citydb.plugin.cli.ResourceIdOption;
+import org.citydb.plugin.cli.SQLSelectOption;
+import org.citydb.plugin.cli.TypeNamesOption;
+import org.citydb.plugin.cli.XMLQueryOption;
+import picocli.CommandLine;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class QueryOption implements CliOption {
+    @CommandLine.ArgGroup(exclusive = false)
+    private TypeNamesOption typeNamesOption;
+
+    @CommandLine.ArgGroup
+    private ResourceIdOption resourceIdOption;
+
+    @CommandLine.ArgGroup
+    private DatabaseIdOption databaseIdOption;
```

Comment:
maybe also add the `databaseIdOption` to the CityGML export command?

Reply:
Added with a6fab8e1663128fbcc905c9c4f9a454b8f370bdd

It should maybe then also be available for a KML/COLLADA/glTF export? But since the corresponding config settings currently do not support this, I would prefer a separate issue on this if we would like to have this option.

Reply:
agree